doc/manual/src/glossary.md

```diff
@@ -21,6 +21,30 @@
 
   [store derivation]: #gloss-store-derivation
 
+- [store derivation output reference]{#gloss-store-derivation-output-reference}
+
+  The string resulting from the concatenation of:
+
+  - The path to a [store derivation](#gloss-store-derivation)
+  - The character `^`
+  - Either:
+    - one of the store derivation's [output names](#gloss-output-name) or
+    - the character `*` denoting all outputs of the derivation
+
+  > **Example**
+  >
+  > Reference to all outputs of a given `stdenv-linux` derivation:
+  >
+  > `/nix/store/xvni94ndxy75v1wdpshk8k0l75nwyc54-stdenv-linux.drv^*`
+
+  > **Example**
+  >
+  > Reference to the `man` output of a given `bash-5.2-p15` derivation:
+  >
+  > `/nix/store/llil5bng8p7203l25mqb5lwdhlaya4d9-bash-5.2-p15.drv^man`
+
```

Comment:
```suggestion

  See also: The store path section of the [installables syntax](@docroot@/command-ref/new-cli/nix.html#store-path), where output references are used.
  
```

It'd be good to link or explain what this syntax is a part of and where it's used.